### ISSUE TITLE
Cannot create pdf

### ISSUE THREAD

**Issue by samarita** [tecnickcom/tcexam#109]
Hello,

I am not able to create pdf document of the tests. The following error appears: 

LET OP: [8] Undefined index: '37' | LINE: 248 | FILE: /hermes/bosnaweb05a/b2459/ipg.xxxx/xxxx.xxx/xxx/admin/code/tce_pdf_results.php
TCPDF ERROR: Some data has already been output, can't send PDF file

Als when I am able to create a test pdf, I get another student name than the one selected.
Can anyone help?

Regards,

**Comment by infohtreasure** [comment]
1. You have to give your cache folder read/write access.

2. Also when I am able to create a test pdf, I get another student name
than the one selected.

Please explain more so as for me to reproduce this error, or are you saying
reffering to test result to PDF?

hope the first suggestion solves that issue.

On Wed, Apr 22, 2015 at 4:52 PM, samarita <notifications@github.com> wrote:

> Hello,
>
> I am not able to create pdf document of the tests. The following error
> appears:
>
> LET OP: [8] Undefined index: '37' | LINE: 248 | FILE:
> /hermes/bosnaweb05a/b2459/ipg.xxxx/xxxx.xxx/xxx/admin/code/tce_pdf_results.php
> TCPDF ERROR: Some data has already been output, can't send PDF file
>
> Als when I am able to create a test pdf, I get another student name than
> the one selected.
> Can anyone help?
>
> Regards,
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tecnickcom/tcexam/issues/109>.
>



-- 
Ifeoluwa Opeyemi O.
*www.ifeoluwaopeyemi.com <http://www.ifeoluwaopeyemi.com>*

Hidden Treasure Computers Limited
*RC894550*
www.htreasure.com
info.htreasure@gmail.com
*+234 803 9254 192*

**Comment by samarita** [comment]
Hello, 

Thank you for responding. 

I will check the authorization on the cache folder. 

With regards to the second issue: after the test has been completed, I
always create a PDF of the test so I can add comments if necessary.
 While creating this document I get the error message. 

Regards, 

On 2015-04-23 11:52, Hideen Treasures wrote: 

> 1. You have to give your cache folder read/write access.
> 
> 2. Also when I am able to create a test pdf, I get another student name
> than the one selected.
> 
> Please explain more so as for me to reproduce this error, or are you saying
> reffering to test result to PDF?
> 
> hope the first suggestion solves that issue.
> 
> On Wed, Apr 22, 2015 at 4:52 PM, samarita <notifications@github.com> wrote:
> 
>> Hello,
>>
>> I am not able to create pdf document of the tests. The following error
>> appears:
>>
>> LET OP: [8] Undefined index: '37' | LINE: 248 | FILE:
>> /hermes/bosnaweb05a/b2459/ipg.xxxx/xxxx.xxx/xxx/admin/code/tce_pdf_results.php
>> TCPDF ERROR: Some data has already been output, can't send PDF file
>>
>> Als when I am able to create a test pdf, I get another student name than
>> the one selected.
>> Can anyone help?
>>
>> Regards,
>>
>> --
>> Reply to this email directly or view it on GitHub
>> <https://github.com/tecnickcom/tcexam/issues/109>.
>>
> 
> -- 
> Ifeoluwa Opeyemi O.
> *www.ifeoluwaopeyemi.com <http://www.ifeoluwaopeyemi.com>*
> 
> Hidden Treasure Computers Limited
> *RC894550*
> www.htreasure.com
> info.htreasure@gmail.com
> *+234 803 9254 192* 
> 
> --
> Reply to this email directly or view it on GitHub [1].

 

Links:
------
[1]
https://github.com/tecnickcom/tcexam/issues/109#issuecomment-95517284

**Comment by infohtreasure** [comment]
I think i get your point now.

when you generate a test report and you want DPF for a particular student
sellecting the student don't work,

if you want to generate PDF for a particular student select the student
name from the user drop down menu, that use to work for me most time.



I hiope that help?

On Thu, Apr 23, 2015 at 1:30 PM, samarita <notifications@github.com> wrote:

>
>
> Hello,
>
> Thank you for responding.
>
> I will check the authorization on the cache folder.
>
> With regards to the second issue: after the test has been completed, I
> always create a PDF of the test so I can add comments if necessary.
> While creating this document I get the error message.
>
> Regards,
>
> On 2015-04-23 11:52, Hideen Treasures wrote:
>
> > 1. You have to give your cache folder read/write access.
> >
> > 2. Also when I am able to create a test pdf, I get another student name
> > than the one selected.
> >
> > Please explain more so as for me to reproduce this error, or are you
> saying
> > reffering to test result to PDF?
> >
> > hope the first suggestion solves that issue.
> >
> > On Wed, Apr 22, 2015 at 4:52 PM, samarita <notifications@github.com>
> wrote:
> >
> >> Hello,
> >>
> >> I am not able to create pdf document of the tests. The following error
> >> appears:
> >>
> >> LET OP: [8] Undefined index: '37' | LINE: 248 | FILE:
> >>
> /hermes/bosnaweb05a/b2459/ipg.xxxx/xxxx.xxx/xxx/admin/code/tce_pdf_results.php
> >> TCPDF ERROR: Some data has already been output, can't send PDF file
> >>
> >> Als when I am able to create a test pdf, I get another student name than
> >> the one selected.
> >> Can anyone help?
> >>
> >> Regards,
> >>
> >> --
> >> Reply to this email directly or view it on GitHub
> >> <https://github.com/tecnickcom/tcexam/issues/109>.
> >>
> >
> > --
> > Ifeoluwa Opeyemi O.
> > *www.ifeoluwaopeyemi.com <http://www.ifeoluwaopeyemi.com>*
> >
> > Hidden Treasure Computers Limited
> > *RC894550*
> > www.htreasure.com
> > info.htreasure@gmail.com
> > *+234 803 9254 192*
> >
> > --
> > Reply to this email directly or view it on GitHub [1].
>
>
>
> Links:
> ------
> [1]
> https://github.com/tecnickcom/tcexam/issues/109#issuecomment-95517284
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tecnickcom/tcexam/issues/109#issuecomment-95570002>.
>



-- 
Ifeoluwa Opeyemi O.
*www.ifeoluwaopeyemi.com <http://www.ifeoluwaopeyemi.com>*

Hidden Treasure Computers Limited
*RC894550*
www.htreasure.com
info.htreasure@gmail.com
*+234 803 9254 192*

**Comment by samarita** [comment]
Hi,

 

I am sorry, but answers or suggestions do not apply to my issue. 

 

I have been using Tcexam for a very long time now and I have never needed to change the permission on any folder.

 

With regards to creating a pdf document of any test the following. This has also worked fine. My guess is that it has something to do with the mysql database. 

 

But thanks for trying.

 

Regards,

**Comment by samarita** [comment]
Hello,

Go to <tests> then to <users>. Select user, select test and click on PDF.

The PDF created is from another student than the one I have selected.

Does anyone know what is going on here?

Regards,

**Comment by Hiddenreasure** [comment]
Please be sure you sure you are using the current version of TCExam?



On Tue, Apr 28, 2015 at 10:05 PM, samarita <notifications@github.com> wrote:

> Hello,
>
> Go to then to . Select user, select test and click on PDF.
>
> The PDF created is from another student than the one I have selected.
>
> Does anyone know what is going on here?
>
> Regards,
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tecnickcom/tcexam/issues/109#issuecomment-97206921>.
>



-- 
Ifeoluwa Opeyemi O.
*www.ifeoluwaopeyemi.com <http://www.ifeoluwaopeyemi.com>*

Hidden Treasure Computers Limited
*RC894550*
www.htreasure.com
info.htreasure@gmail.com
*+234 803 9254 192*

**Comment by samarita** [comment]
I made sure of that before I updated my comment.

**Comment by Hiddenreasure** [comment]
Hello samita. Now I'm able to reproduce your error message.

In my own case I have 30questions setup for an exam when generating PDF. I
get the message.


Here is how I solve the problem.

1. I click on list to view all the questions at once and I noticed some of
the questions did not format well.

2. I scroll to the first question where the improper formating begins and I
modify it to correct the error. In my case I use (olist) and I change it.

3. That corrected all other as well. Been the only one that affect all and
I was able to PDF.

4. In your case. If you have perform the said test make unavailable the
question with error and create another to correct the error.

Hope This help?

On Tuesday, April 28, 2015, samarita <notifications@github.com> wrote:
> I made sure of that before I updated my comment.
>
> —
> Reply to this email directly or view it on GitHub.<
https://ci6.googleusercontent.com/proxy/xmvxacF-LOowTZeLj0z2L5MJQbLeaDdVAqAwYSIoDJwPAaVK0P0e5zxSzcCiwsmmGwPPP1bOMuz7XNJjrQrOfeq4TR-m6dGU0bqQC0PPc1aWm68SS8yO8NFdiB20TrY530OzPbvpwGXQVWa5SpNwZJ93zkC3iw=s0-d-e1-ft#https://github.com/notifications/beacon/AHneSzvbr0l7NKopMvuGErSEESSV_eyaks5oEAKzgaJpZM4EGLKM.gif
>

-- 
Ifeoluwa Opeyemi O.
*www.ifeoluwaopeyemi.com <http://www.ifeoluwaopeyemi.com>*

Hidden Treasure Computers Limited
*RC894550*
www.htreasure.com
info.htreasure@gmail.com
*+234 803 9254 192*
